### ISSUE TITLE
Add parameter to specify how a task steps with one step should be rendered

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/dita-ot#4072]
Based on this discussion: https://www.oxygenxml.com/forum/viewtopic.php?p=68675#p68675

Suppose I have a DITA task with a steps containing only one step. Both the HTML and PDF based outputs no longer present a number before the single step. Maybe a parameter could be added to control this behavior.

**Comment by robander** [comment]
Just noting that this has been the DITA-OT default since version 1.0 (the "no longer" above makes it sound like this is a change).

Over the past couple of years we have generally suggested plugins for changes like this rather than adding a lot of very specific build parameter - this one seems like that kind of change, where we do not want to add a parameter for such a specific style change. The plugin override for this is not very complex (compared to most style plugins, of course it still requires a little bit of XSLT).

**Comment by raducoravu** [comment]
Right, I also recommended the end user a CSS based customization which works at least for the HTML based outputs (and for our PDF CSS based output as well).
